### ISSUE TITLE
IfUnlessModifier versus MultilineIfModifier

### ISSUE THREAD

**Issue by kddeisz** [rubocop-hq/rubocop#7110]
I have the following code structure:

```ruby
foobar =
  if foobar
    foobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobar
  end
```

This fails `Style/IfUnlessModifier` because technically the body of the if statement and the predicate could fit on one line. So I change it to:

```ruby
foobar =
  foobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobar if foobar
```

But that fails `Style/MultilineIfModifier` because technically it's multiline. I think probably both of those should pass, but I'd be fine with either one.

## RuboCop version

```
$ [bundle exec] rubocop -V
0.70.0 (using Parser 2.6.3.0, running on ruby 2.6.3 x86_64-darwin18)
```

**Comment by buehmann** [comment]
```ruby
foobar =
  foobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobar if foobar
```
is not equivalent to your original code: Here the condition applies to the assignment; in the orignal version it applies to the right-hand side only.

This would be equivalent:
```ruby
foobar =
  (foobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobarfoobar if foobar)
```
And both cops are happy with it. (I am not; so I'd probably rewrite this.)

**Comment by kddeisz** [comment]
Interesting - I'll close this, thanks for the feedback.
